nixos/tests/all-tests.nix

```diff
@@ -120,7 +120,7 @@ in
   handbrake = handleTestOn ["x86_64-linux"] ./handbrake.nix {};
   haproxy = handleTest ./haproxy.nix {};
   hardened = handleTest ./hardened.nix {};
-  hibernate = handleTest ./hibernate.nix {};
+  hibernate = handleTestOn ["x86_64-linux"] ./hibernate.nix {};
```

Comment:
could you comment why with a some sort of reference why it is only on x86_64-linux?